InstabugSample/src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { Switch, StyleSheet, Text, View } from 'react-native';
+import Instabug, { BugReporting } from 'instabug-reactnative';
+
+import Button from '../components/Button';
+import ColorButton from '../components/ColorButton';
+import Screen from '../components/Screen';
+import Section from '../components/Section';
+
+const colors = ['crimson', 'olivedrab', 'cornflowerblue', 'gold'];
+
+function SettingsScreen() {
+  const [isLightMode, setIsLightMode] = useState(true);
+  const [currentColor, setCurrentColor] = useState('');
+
+  const toggleColorTheme = _isLightMode => {
+    setIsLightMode(_isLightMode);
+    Instabug.setColorTheme(_isLightMode ? Instabug.colorTheme.light : Instabug.colorTheme.dark);
```

Comment:
**suggestion:** Staying consistent with the **order** of `setState` and `Instbug` methods, would be nice 🤔